src/components/node/stanford-news/page-display.tsx

```diff
@@ -141,7 +141,7 @@ const StanfordNews = async ({node, ...props}: {node: NodeStanfordNews}) => {
       )}
 
       {node.suNewsComponents && (
-        <div className="w-full [&_p]:mx-auto lg:[&_p]:max-w-800">
+        <div className="[&>.wysiwyg]:max-w-800">
```

Comment:
Could you apply the class directly to the `<Paragraph` using clsx? 
`<Paragraph className={clsx({"foo-bar": paragraph.__typename === "FooBarParagraph"})} />`